res/controllers/DJ TechTools-MIDI Fighter Twister-scripts.js

```diff
@@ -23,6 +23,12 @@ var MidiFighterTwister = {
             engine.makeConnection("[Channel2]", "pregain", function(value) {
                 midi.sendShortMsg(cc, 0x03, linearize(value));
             }),
+            engine.makeConnection("[Channel1]", "volume", function(value) {
+                midi.sendShortMsg(cc, 0x08, linearize(value) * 2);
+            }),
+            engine.makeConnection("[Channel2]", "volume", function(value) {
+                midi.sendShortMsg(cc, 0x0B, linearize(value) * 2);
+            }),
```

Comment:
What about the PFL button? I think the LED below the volume knob should show the state of that.